include/ola/client/ClientArgs.h

```diff
@@ -90,9 +90,9 @@ struct SendDMXArgs {
   /**
    * @brief Create a new SendDMXArgs object
    */
-  explicit SendDMXArgs(GeneralSetCallback *callback)
+  explicit SendDMXArgs(GeneralSetCallback *_callback)
```

Comment:
Not for option structs. We shouldn't force the clients to do:

options.m_priority = X;

but rather:

options.priority = X;

I don't mind what the constructor args are called. I would have gone with callback_arg but _callback works as well.